splash/qtrender_lua.py

```diff
@@ -758,6 +758,36 @@ def callback(reply):
     def autoload_reset(self):
         self.tab.autoload_reset()
 
+    def _check_mouse_coordinantes(self, x, y):
+        try:
+            assert isinstance(x, int) or isinstance(x, float)
+            assert isinstance(y, int) or isinstance(y, float)
+        except AssertionError:
+            raise ScriptError({
+                "argument": "x, y",
+                "message": "splash mouse event x, y coordinates must be int or float "
```

Comment:
there are no ints and floats in Lua; what about changing it to "splash mouse even x coordinate is not a number"?